L/libflame/build_tarballs.jl

```diff
@@ -0,0 +1,89 @@
+name = "libflame"
+
+using BinaryBuilder, Pkg
+
+version = v"5.2.0"
+
+# Collection of sources required to build lapack
+sources = [
+    GitSource("https://github.com/flame/libflame",
+              "646ed9075cc45aeffb53632a3ec88defb423fefa"),
+    DirectorySource("bundled"),
+]
+
+# Bash recipe for building across all *64-bit* platforms
+# We try to maintain consistency with the blis Yggdrasil build scripts.
+
+script = raw"""
+cd $WORKSPACE/srcdir/libflame
+
+# We might need newer `config.guess`` and `config.sub` files
+update_configure_scripts
+
+extra_flags=" --disable-static-build "
+
+if [[ "${target}" == *-apple-* ]]; then
+    extra_flags=" --enable-static-build "
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/mac-update-makefile.patch
+fi
+
+if [[ "${target}" == *-w64-mingw32* ]]; then
+    extra_flags+=" --enable-windows-build "
+    # disable time & clock functions on windows (mingw): sys/times.h is missing
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows-remove-time.patch
+    # update the make build scripts for windows cross-compiling with gcc:
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows-update-build.patch
+fi
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT="-L${libdir} -lblastrampoline-5"
+else
+    LBT="-L${libdir} -lblastrampoline"
+fi
+
+# - Compile and build a LAPACK compatibility layer with --enable-lapack2flame
+# - If a static library is not needed, use --disable-static-build
+# - Enable a dynamic build with --enable-dynamic-build
+./configure \
+    --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --enable-multithreading=openmp \
+    --enable-lapack2flame \
+    --enable-dynamic-build \
+    --enable-max-arg-list-hack \
+    --disable-autodetect-f77-ldflags --disable-autodetect-f77-name-mangling \
+    $extra_flags
+
+make -j${nproc} BLAS_LIB="${LBT}"
+make install
+
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+filter!(p -> arch(p) != "i686", platforms)
+filter!(p -> arch(p) != "armv7l", platforms)
+filter!(p -> arch(p) != "armv6l", platforms)
+filter!(p -> arch(p) != "powerpc64le", platforms)
+
+# Enable the following line if trying to link against Fortran code:
+# platforms = expand_gfortran_versions(platforms)
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), compat="5.9.0"),
```

Comment:
Just a note, doing a compat to 5.9.0 here will limit this to Julia 1.11+ I think (LBT 5.9.0 is in 1.12 for sure, and was discussed as a backport for 1.11, but I don't think that has happened yet).

Julia 1.9.1+ and 1.10 had libblastrampoline 5.8 in it, and the only difference should just be new symbols for the new LAPACK, so I think it should be safe to limit to 5.8 instead.

Reply:
Thanks